TypeScript Files/src/addArrayTimeout.ts

```diff
@@ -0,0 +1,30 @@
+import getArrayObject from './getArrayObject';
+import { Myconstants, MyStrings, Iobjectarray } from './enum.d'
+
+let addArrayTimeout = (finalArrayObject:Required<Iobjectarray[]>,del :number, valuesarray : number[]) :(Required<Iobjectarray[]> | Promise<unknown>)[] => {
```

Comment:
it takes the required Iobjectarray which is an array where the adding is done, the del is a number used in the timeout function for to set the delay and the valuesarray[] is the array of the order number added and this array should be added to the Iobjectarray array..
it will return two values inside an array where the first value of an array is the value of the promise and the second value is the new Iobjectarray which is the old array plus the valuesarray